cmd/protoc-gen-go-grpc/go.mod

```diff
@@ -0,0 +1,8 @@
+module google.golang.org/grpc/cmd/protoc-gen-go-grpc
+
+go 1.9
+
+require (
+	github.com/golang/protobuf v1.4.2
```

Comment:
None of the files in this module use this dependency. Is it still needed?

Reply:
It is not. Removed.

Reply:
Related question. Is it intended that `cmd/protoc-gen-go-grpc` should be its own module? This is not the case for `cmd/protoc-gen-go`. I'm asking because multi-module repos has certain complications, but perhaps it is necessary for this case.

Reply:
Yes, this is intentional.  We don't want to release this along with grpc itself.  And the initial version will be v0.0, which means we may (and intend to) make backward-incompatible changes before performing a release at v1.0.  (We intend to require the inclusion of the `Unimplemented____Service` struct unless a command-line flag is specified.)